Destination/Destination.php

```diff
@@ -3,15 +3,25 @@
 namespace Zenstruck\BackupBundle\Destination;
 
 use Psr\Log\LoggerInterface;
+use \Countable;
+use \IteratorAggregate;
 
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
-interface Destination
+interface Destination extends Countable, IteratorAggregate
```

Comment:
I do not know if you want me to remove Countable, IteratorAggregate, I have left those. Still think that it is not wrong approach - take per example Twig_Node implementation... It is common practice for aggregating object to implement iterator.